modules/browser.js

```diff
@@ -0,0 +1,35 @@
+// -----------------------------------------------------------------------------------------
+// #region Imports
+// -----------------------------------------------------------------------------------------
+
+const echo = require("./echo");
+const fs = require("fs");
+const os = require("os");
+const shell = require("shelljs");
+
+// #endregion Imports
+
+// -----------------------------------------------------------------------------------------
+// #region Public Members
+// -----------------------------------------------------------------------------------------
+
+const browser = {
+    /**
+     * Opens the supplied URL in the system's default browser
+     * @param {string} url absolute url
+     */
+    open(url) {
+        const command = os.platform() === "win32" ? "start" : "open";
```

Comment:
you can just use `git web--browse` here instead of relying on platform detection directly; it's a utility git command to open the default web browser.

Reply:
> you can just use `git web--browse` here instead of relying on platform detection directly; it's a utility git command to open the default web browser.

Ah, an interesting suggestion! Given we certainly can rely on git being here, I'll take the recommendation. It definitely provides more cross-platform support too. Thanks!